packages/relay/src/lib/eth.ts

```diff
@@ -782,10 +783,15 @@ export class EthImpl implements Eth {
         if (result?.type === constants.TYPE_TOKEN) {
           this.logger.trace(`${requestIdPrefix} Token redirect case, return redirectBytecode`);
           return EthImpl.redirectBytecodeAddressReplace(address);
-        }
-        else if (result?.type === constants.TYPE_CONTRACT) {
+        } else if (result?.type === constants.TYPE_CONTRACT) {
           if (result?.entity.runtime_bytecode !== EthImpl.emptyHex) {
-            return result?.entity.runtime_bytecode;
+            const prohibitedOpcodes = ['CALLCODE', 'DELEGATECALL', 'SELFDESTRUCT', 'SUICIDE'];
+            const opcodes = asm.disassemble(result?.entity.runtime_bytecode);
+            const hasProhibitedOpcode = opcodes.filter(opcode => prohibitedOpcodes.indexOf(opcode.opcode.mnemonic) > -1).length > 0;
+            if (!hasProhibitedOpcode) {
```

Comment:
Add a check for `runtime_bytecode === '0x'` as right now without complete traceability migration some contracts may report this but have valid code and for the mean time should still call the consensus node

Reply:
It's handled [here](https://github.com/hashgraph/hedera-json-rpc-relay/pull/1170/files/25d48c94c373db3528cf48500b3c8e46bfafee18#diff-dd8c7990f7764960d05dcb4c1b5ae7eee48b425b1b83d6dcc7e7ccc442330ef0R787), right :thinking: We don't need an additional check.

Reply:
Ah good point. Okay no-op for this comment then :)